data/data/gcp/cluster/iam/main.tf

```diff
@@ -19,3 +19,9 @@ resource "google_project_iam_member" "worker-storage-admin" {
   role    = "roles/storage.admin"
   member  = "serviceAccount:${google_service_account.worker-node-sa.email}"
 }
+
+resource "google_project_iam_member" "worker-artifact-registry-admin" {
+  project = var.project_id
+  role    = "roles/artifactregistry.admin"
```

Comment:
We are currently exploring ways to assign less permissions. These roles grant too many unused roles that come with risks. 

/hold